### PR TITLE
Refactor ArticleCard to be accessible, clear console errors related to it

### PR DESCRIPTION
**Pull request by sandylcruz**
## Preview

* [new index page](https://blog-git-sandy-console-gravitational.vercel.app/blog/)
* [current live index](https://teleport-blog-next.vercel.app/blog/)

## Issue

* Console warnings about having  an `<a>` as a descendant of another `<a>`
* This was due to using a `<Link>` as the outer container of `<ArticleCard>` while also having an inner link for `<Tags>` at the bottom of the container

## Solution
* Instead of having a`<Link>` as the parent component of `<Tags>`, I made them sibling components
* Maintained the functionality of the entire card being clickable using pseudo-selectors (more information [here](https://inclusive-components.design/cards/))
* All links are now tab-focusable by screen readers

## Before

Screen Shot 2022-03-09 at 4 02 58 PM

## After

Screen Shot 2022-03-10 at 2 41 14 PM

### CODE REVIEW

File: components/ArticleCard/ArticleCard.tsx [hunk 1]
```diff
@@ -21,52 +21,96 @@ export default function ArticleCard({ meta }: ArticleCardProps) {
   const parsedDate = getParsedDate(meta.frontmatter.date);
 
   return (
-    <StyledCard href={meta.uri} flexDirection={!!image ? "row" : "column"}>
-      {!!image && (
-        <StyledWrapperImage>
+    <OuterContainer style={{ flexDirection: image ? "row" : "column" }}>
+      {image && (
+        <ImageContainer>
```
Comment (C-STYR) [inline components/ArticleCard/ArticleCard.tsx:26]: How would you feel about locally defined components dropping all the extraneous descriptive elements where possible: `DateContainer` becomes `Date`, `ArticleTitleContainer` becomes `Title`, etc.  So as not to confuse `Image` with `NextImage`, we could keep `ImageContainer` or `ImageWrapper`.  Something to think about for future development, not necessarily for this page.

File: components/ArticleCard/ArticleCard.tsx [hunk 2]
```diff
@@ -21,52 +21,96 @@ export default function ArticleCard({ meta }: ArticleCardProps) {
...
           <NextImage
             src={image}
-            alt="article image"
+            alt={meta.frontmatter.logo.alt}
```
Comment (C-STYR) [inline components/ArticleCard/ArticleCard.tsx:29]: Nice.

**Review by C-STYR — APPROVED** [review]

File: components/ArticleCard/ArticleCard.tsx [hunk 3]
```diff
@@ -21,52 +21,96 @@ export default function ArticleCard({ meta }: ArticleCardProps) {
   const parsedDate = getParsedDate(meta.frontmatter.date);
 
   return (
-    <StyledCard href={meta.uri} flexDirection={!!image ? "row" : "column"}>
-      {!!image && (
-        <StyledWrapperImage>
+    <OuterContainer style={{ flexDirection: image ? "row" : "column" }}>
```
[the remark this answers is not in the document]
Comment (sandylcruz) [reply components/ArticleCard/ArticleCard.tsx:24]: I thought it'd look cleaner that way but the double bang should be there. Changing

File: components/ArticleCard/ArticleCard.tsx (hunk 1 above)
Comment (sandylcruz) [reply components/ArticleCard/ArticleCard.tsx:26]: Makes sense, it was starting to look like a lot of text. Updating now.

**Review by C-STYR — APPROVED** [review]

**Pull request merged**